src/output/icons.rs

```diff
@@ -52,41 +52,131 @@ lazy_static! {
         m.insert(".atom", '\u{e764}'); // 
         m.insert(".bashprofile", '\u{e615}'); // 
         m.insert(".bashrc", '\u{f489}'); // 
-        m.insert(".git", '\u{f1d3}'); // 
+        m.insert(".config", '\u{e5fc}'); // 
+        m.insert(".clang-format", '\u{e615}'); // 
+        m.insert(".doom.d", '\u{e779}'); // 
+        m.insert(".emacs.d", '\u{e779}'); // 
+        m.insert(".git", '\u{e5fb}'); // 
         m.insert(".gitattributes", '\u{f1d3}'); // 
         m.insert(".gitconfig", '\u{f1d3}'); // 
-        m.insert(".github", '\u{f408}'); // 
+        m.insert(".github", '\u{e5fd}'); // 
         m.insert(".gitignore", '\u{f1d3}'); // 
         m.insert(".gitmodules", '\u{f1d3}'); // 
+        m.insert(".gitlab-ci.yml", '\u{f296}'); // 
+        m.insert(".htaccess", '\u{e615}'); // 
+        m.insert(".htpasswd", '\u{e615}'); // 
+        m.insert(".node_repl_history", '\u{e718}'); // 
+        m.insert(".python_history", '\u{e606}'); // 
+        m.insert(".release.toml", '\u{e7a8}'); // 
+        m.insert(".rustfmt.toml", '\u{e7a8}'); // 
         m.insert(".rvm", '\u{e21e}'); // 
+        m.insert(".ssh", '\u{f023}'); // 
+        m.insert(".vim", '\u{e62b}'); // 
+        m.insert(".viminfo", '\u{e62b}'); // 
         m.insert(".vimrc", '\u{e62b}'); // 
         m.insert(".vscode", '\u{e70c}'); // 
+        m.insert(".xinitrc", '\u{e615}'); // 
+        m.insert(".Xauthority", '\u{e615}'); // 
+        m.insert(".Xresources", '\u{e615}'); // 
         m.insert(".zshrc", '\u{f489}'); // 
+        m.insert(".zsh_history", '\u{e615}'); // 
         m.insert("Cargo.lock", '\u{e7a8}'); // 
+        m.insert("Cargo.toml", '\u{e7a8}'); // 
+        m.insert("a.out", '\u{f489}');
+        m.insert("authorized_keys", '\u{e60a}'); // 
+        m.insert("autostart", '\u{f489}'); // 
         m.insert("bin", '\u{e5fc}'); // 
+        m.insert("bspwmrc", '\u{e615}'); // 
+        m.insert("composer.json", '\u{e608}'); // 
+        m.insert("composer.lock", '\u{e608}'); // 
         m.insert("config", '\u{e5fc}'); // 
+        m.insert("config.ac", '\u{e615}'); // 
+        m.insert("config.el", '\u{e779}'); // 
+        m.insert("config.m4", '\u{e615}'); // 
+        m.insert("config.mk", '\u{e615}'); // 
+        m.insert("configure", '\u{f489}'); // 
+        m.insert("contributing", '\u{e60a}'); // 
```

Comment:
example

Reply:
Not very useful in my opinion since a lot of contributing files are `.md` or the like, so since the icon is bad may remove it? Worst case we can still add a better one later